### PR TITLE
Fixed a typo.

### LINKED ISSUE #214: Typo(s) in /roles/elasticsearch/templates /elasticsearch.yml.j2
Hi,

 there are some typos in lines 6-10 (ealsticsearch instead of elasticsearch)
 
`{% if ealsticsearch_http_publish_host is defined %}
http.publish_host: "{{ ealsticsearch_http_publish_host }}"
{% endif %}
{% if ealsticsearch_http_publish_port is defined %}
http.publish_port: {{ ealsticsearch_http_publish_port }}`

### LINKED ISSUE #217: Typo in Variable present in whole code for ealsticsearch_http_publish_host
https://github.com/NETWAYS/ansible-collection-elasticstack/blob/22c2a9dd51b6f5a58dfa3c985245b2fd70deb368/docs/role-elasticsearch.md?plain=1#L34

*ealsticsearch_http_publish_host* Typo is present in the whole code like tasks and templates.

### PR DESCRIPTION
**Pull request by widhalmt**
Thanks @xtruthx and 
@frankhetterich for finding it.

fixes #217
fixes #214

### CODE REVIEW

**Comment by widhalmt** [comment]
Checks failing due to problems in GitHub test infrastructure or being too excessive with tests.

File: docs/role-elasticsearch.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Role Variables
...
 * *elasticsearch_bootstrap_pw*: Bootstrap password for Elasticsearch (Default: `PleaseChangeMe`)
 * *elasticsearch_disable_systemcallfilterchecks*: Disable system call filter checks. This has a security impact but is necessary on some systems. Please refer to the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/_system_call_filter_check.html) for details. (default: `false`)
-* *ealsticsearch_http_publish_host*: (String) The network address for HTTP clients to contact the node using sniffing. Accepts an IP address, a hostname, or a special value. (default: `not set`)
+* *elasticsearch_http_publish_host*: (String) The network address for HTTP clients to contact the node using sniffing. Accepts an IP address, a hostname, or a special value. (default: `not set`)
 * *ealsticsearch_http_publish_port*: (integer) The port of the HTTP publish address. Configure this setting only if you need the publish port to be different from http.port. (default: `not set`)
```
Comment (Donien) [inline docs/role-elasticsearch.md:35]: Another `ealstic`

File: roles/elasticsearch/templates/elasticsearch.yml.j2 [hunk 2]
```diff
@@ -3,8 +3,8 @@ path.data: {{ elasticsearch_datapath }}
 path.logs: {{ elasticsearch_logpath }}
 cluster.name: "{{ elasticsearch_clustername }}"
 network.host: ["_local_","_site_"]
-{% if ealsticsearch_http_publish_host is defined %}
-http.publish_host: "{{ ealsticsearch_http_publish_host }}"
+{% if elasticsearch_http_publish_host is defined %}
+http.publish_host: "{{ elasticsearch_http_publish_host }}"
 {% endif %}
 {% if ealsticsearch_http_publish_port is defined %}
 http.publish_port: {{ ealsticsearch_http_publish_port }}
```
Comment (Donien) [inline roles/elasticsearch/templates/elasticsearch.yml.j2:10]: Another two `ealstic`s

**Review by Donien — CHANGES_REQUESTED** [review]
I've found some more `ealstic`

**Review by Donien — APPROVED** [review]
Looks good to me now 👍🏻

**Review by dnssmr — APPROVED** [review]

**Pull request merged**